src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -1116,7 +1118,7 @@ class LicenceService(
     isDueToBeReleasedInTheNextTwoWorkingDays = releaseDateService.isDueToBeReleasedInTheNextTwoWorkingDays(this),
   )
 
-  private fun assertCaseIsEligible(licenceId: Long, nomisId: String?) {
+  internal fun assertCaseIsEligible(licenceId: Long, nomisId: String?) {
```

Comment:
To allow me to verify in a test,... I try to avoid changing production code for test convenience, at least it's not public

Reply:
could actually just test for this eligibilityService.isEligibleForCvl(prisoner)